internal/pkg/composable/providers/kubernetes/pod.go

```diff
@@ -161,9 +161,9 @@ func (p *pod) emitRunning(pod *kubernetes.Pod) {
 				annotations, _ := ann.(mapstr.M)
 				hints := utils.GenerateHints(annotations, "", p.config.Prefix)
 				if len(hints) > 0 {
-					p.logger.Errorf("Extracted hints are :%v", hints)
+					p.logger.Debugf("Extracted hints are :%v", hints)
```

Comment:
Can those go to info? 
Correct me if I am wrong but lets make them appear everytime to help troubleshooting later

Reply:
Info level is logged always...So these will be logged for every Pod's event...this is quite noisy. For debugging efforts we can just ask for debug logs :) .